LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
```

Comment:
Not sure why our linter doesn't catch this (probably because the linter runs against the code on master instead of the commit on the branch) but we'd also have to update the headers in the code themselves.

```
➜  hypersdk git:(goodbye-2024) grep -r 'Copyright (C) 2024' . -l | wc -l
     401
```